src/systems/MoveableSystem.js

```diff
@@ -21,15 +21,18 @@ function MoveableSystem(entity, dX, dY) {
 
 	if (map.isTileOccupied(newX, newY)) {
 		let occupiedBy = map.getEntityAt(newX, newY);
-		Logger(`${entity.getName()} tries to move into ${newX}, ${newY} but there is a ${occupiedBy.getName()} there.`);
+
+		if (entity.hasComponent('Player')) {
+			Logger(`${entity.getName()} tries to move into ${newX}, ${newY} but there is a ${occupiedBy.getName()} there.`);
+		}
 
 		//TODO Don't make enemies attack eachother when they are in the way.
 		BattleSystem(entity, occupiedBy);
 
 		return;
 	}
 
-	if (map.isTileSolid(newX, newY) && !entity.hasComponent('PassThroughSolids')) {
+	if (map.isTileSolid(newX, newY) && !entity.hasComponent('PassThroughSolids') && entity.hasComponent('Player')) {
```

Comment:
Again, I think this will make everyone be able to pass through walls except for the player.